PracticalTasksAboutSpringDataJpa/src/main/java/com/example/PracticalTasksAboutSpringDataJpa/repository/projections/CustomerWithAddress.java

```diff
@@ -0,0 +1,13 @@
+package com.example.PracticalTasksAboutSpringDataJpa.repository.projections;
+
+public interface CustomerWithAddress {
```

Comment:
rownie dobrze mozesz zaczac rozszerzac interfejs customerSummary bo 3 jednakowe metody kopiujesz z miejsca w miejsce i po co. Lub ewentualnei implementowac dwa interfejsy ale to bez sensu akurat w tym przypadku